tests/_test_mscolab/test_server.py

```diff
@@ -234,19 +235,27 @@ def test_get_all_changes(self):
             operation, token = self._create_operation(test_client, self.userdata)
             fm, user = self._save_content(operation, self.userdata)
             fm.save_file(operation.id, "content2", user)
+            all_changes = fm.get_all_changes(operation.id, user)
```

Comment:
This shouldn't be necessary to get all changes, the line below this one uses the API endpoint to get them so they should be extractable from `response`.

Reply:
Yes, that is expected since the `all_changes` variable is no longer defined then. But the same data is already contained in `response` and should be taken from there in this test. The asserts need to be adapted for that.

Reply:
I've talked to @ReimarBauer and we came to the conclusion that this is not a blocker for this PR. Other tests in test_server.py could be improved in this regard as well and it can be addressed in the future.